src/sentry/mediators/external_requests/select_requester.py

```diff
@@ -34,7 +34,7 @@ def call(self):
 
     def _build_url(self):
         domain = urlparse(self.sentry_app.webhook_url).netloc
-        url = u'https://{}{}'.format(domain, self.uri)
+        url = u'http://{}{}'.format(domain, self.uri)
```

Comment:
Why the change? Should you be getting the protocol from the webhook URL instead?

Reply:
Just to test it out locally. Yea, this should be based on their URL.